packages/evaluatorq-py/src/evaluatorq/table_display.py

```diff
@@ -65,18 +76,23 @@ def create_summary_display(results: EvaluatorqResult, console: Console) -> Table
     return table
 
 
-def calculate_evaluator_averages(results: EvaluatorqResult) -> dict[str, Any]:
+def calculate_evaluator_averages(
+    results: EvaluatorqResult,
+) -> EvaluatorAverages:
     """
     Calculate averages for evaluator scores across all data points.
 
     Returns:
-        Dictionary with job_names, evaluator_names, and averages
+        EvaluatorAverages with:
+        - job_names: sorted list of job names
+        - evaluator_names: sorted list of evaluator names
+        - averages: dict mapping evaluator_name -> job_name -> (display_value, style)
     """
-    all_job_names: Set[str] = set()
-    all_evaluator_names: Set[str] = set()
+    all_job_names: set[str] = set()
+    all_evaluator_names: set[str] = set()
 
     # Store all scores per evaluator per job
-    scores_by_evaluator_and_job: Dict[str, Dict[str, List[float | bool | str]]] = (
+    scores_by_evaluator_and_job: dict[str, dict[str, list[float | bool | str]]] = (
```

Comment:
I appreciate the dedication to typehinting, but any chance we can make this less nested or use pydantic models/typeddicts to simplify this?